packages/templates/package.json

```diff
@@ -16,6 +16,7 @@
     "@web-stories-wp/tracking": "*"
   },
   "devDependencies": {
-    "@web-stories-wp/stickers": "*"
+    "@web-stories-wp/stickers": "*",
+    "@web-stories-wp/date": "*"
```

Comment:
Not actually being used currently?

Reply:
just used for tests. kept all the dates as pure strings in the template data